scss/_layouts_application-panels.scss

```diff
@@ -65,6 +113,11 @@
     padding-top: $spv--large;
   }
 
+  .p-panel__controls-tagged-logo {
```

Comment:
It would be good if we can avoid this. Adding new class name is fine, but hopefully it should not be necessary to have new class name for controls because of new logo.

Current styles of `.p-panel__controls` are not ideal actually, so maybe we can improve those to work fine with old and new logo. (Making it work with new logo would be a priority, I guess it's fine if controls may be slightly misaligned with old logo - we likely don't have a lot apps depending on it).